content/implementations/ruby/ruby-on-rails.md

```diff
@@ -163,7 +163,8 @@ default: --format pretty
 html: --format html --out features.html
 ```
 
-Remember that you need `AUTOFEATURE=true` for autospec to include Cucumber Features. See [Running Features](/configuration/#running-features) and [Autotest Integration](/wiki/autotest-integration) for more info.
+Remember that you need `AUTOFEATURE=true` for autospec to include Cucumber Features. See [Running Features](/running/) 
```

Comment:
lowercase for "Cucumber Features" to "Cucumber features".
lowercase for "Autotest Integration" to "autotest integration" (unless Autotest is a product name? then it should be "Autotest integration"?)

Reply:
Unsure bout Autotest or autotest. I'll lowercase it for now.